nbinteract/figure.py

```diff
@@ -0,0 +1,259 @@
+"""
+Methods of the Figure and Plot class to create more complex plots.
+"""
+
+import numpy as np
+import bqplot as bq
+import collections
+import ipywidgets as widgets
+import itertools
+import functools
+import logging
+import toolz.curried as tz
+from IPython.display import display
+from . import util
+from . import plotting
```

Comment:
Looks like a lot of these imports aren't used (`numpy`, `collections`, `itertools`, `functools`, and `logging`). Think you can take them out?

(I use https://github.com/SublimeLinter/SublimeLinter with https://github.com/SublimeLinter/SublimeLinter-flake8 to automatically catch these.)

Reply:
done!